src/basic/missing.h

```diff
@@ -87,14 +86,6 @@ struct sockaddr_vm {
 #define IP_FREEBIND 15
 #endif
 
-#ifndef OOM_SCORE_ADJ_MIN
-#define OOM_SCORE_ADJ_MIN (-1000)
-#endif
-
-#ifndef OOM_SCORE_ADJ_MAX
-#define OOM_SCORE_ADJ_MAX 1000
-#endif
-
```

Comment:
Isn't it used in parse-util.c and process-util.c?

Reply:
These are defined in linux/oom.h and the header is already included.

Reply:
It was added in v2.6.35-5872-ga63d83f427. I guess that's old enough.